sdk/servicebus/Azure.Messaging.ServiceBus/tests/Core/ConnectionStringParserTests.cs

```diff
@@ -6,7 +6,7 @@
 using Azure.Messaging.ServiceBus.Core;
 using NUnit.Framework;
 
-namespace Azure.Messaging.ServiceBus.Tests.Client
+namespace Azure.Messaging.ServiceBus.Tests.Core
```

Comment:
Orthogonal, but it took me a bit to find this when looking since it wasn't in the same folder structure as the class being tested.

Reply:
Hmm I thought we had a Client folder for src as well.

Reply:
We do, but the parser was in the Core folder.    I don't feel particularly strongly where it hangs out, just that they live in the same place so that the simpletons like me can find them.  😄

Reply:
Ah I see what you mean.